### ISSUE TITLE
membrane_h264_plugin - negative steps are not supported in String.slice/2

### ISSUE THREAD

**Issue by wkozyra95** [membraneframework/membrane_core#709]
I'm getting a warning (but everything seems to work fine)

```
warning: negative steps are not supported in String.slice/2, pass 1..-1//1 instead
  (elixir 1.16.0) lib/string.ex:2368: String.slice/2
  (membrane_h264_plugin 0.9.1) lib/membrane_h264_plugin/parser/nalu_parser/scheme_parser.ex:210: Membrane.H264.Parser.NALuParser.SchemeParser.parse_field/3
  (membrane_h264_plugin 0.9.1) lib/membrane_h264_plugin/parser/nalu_parser/scheme_parser.ex:104: anonymous fn/3 in Membrane.H264.Parser.NALuParser.SchemeParser.do_parse_with_scheme/4
  (elixir 1.16.0) lib/enum.ex:2528: Enum."-reduce/3-lists^foldl/2-0-"/3
  (membrane_h264_plugin 0.9.1) lib/membrane_h264_plugin/parser/nalu_parser/scheme_parser.ex:91: Membrane.H264.Parser.NALuParser.SchemeParser.parse_with_scheme/4
  (membrane_h264_plugin 0.9.1) lib/membrane_h264_plugin/parser/nalu_parser.ex:67: Membrane.H264.Parser.NALuParser.parse/4
```

when running e.g. https://github.com/membraneframework/membrane_video_compositor_plugin/blob/master/examples/lib/dynamic_outputs.exs

I have not tested it yet, but it's apparently specific to elixir 1.16.

**Comment by bartkrak** [comment]
I found that this problem is fixed in the never version (v0.10.0). Only difference is added h265 support, so you should be able to upgrade easily. Keep in mind that package name has changed, so you have to go from 
`{:membrane_h264_plugin, "~> 0.9.1"}`
to
`{:membrane_h26x_plugin, "~> 0.10.0"}`

**Comment by wkozyra95** [comment]
Thanks, confirm it is resolved
